selfdrive/car/nissan/values.py

```diff
@@ -30,21 +30,21 @@ class CAR:
   ALTIMA = "NISSAN ALTIMA 2020"
 
 
-NISSAN_HARNESS_PARTS = [HarnessPart.harness_box, HarnessPart.rj45_cable, HarnessPart.long_obdc_cable, HarnessPart.usbc_coupler]
+NISSAN_PARTS: List[CarPart] = [CarPart.harness_box, CarPart.rj45_cable_7ft, CarPart.long_obdc_cable, CarPart.usbc_coupler, CarPart.mount, CarPart.right_angle_obd_c_cable_1_5ft]
```

Comment:
same, we can just add long usb c, coupler (car docs bot should catch changing parts lists). unless you think it's better to list everything to be explicit

Reply:
since we have so few exceptions (OBD II, J533 and Nissan), I think it is worth being explicit